src/movies.js

```diff
@@ -1,3 +1,6 @@
+
+// Gizem Wanka && Aleksandra Porada
+
 // Iteration 1: All directors? - Get the array of all directors.
 
 function getAllDirectors(arrayOfMovies) {
```

Comment:
Although a perfectly valid solution, the purpose of this iteration was to try the .map() array method to get you used to it! 💡